app/views/api_docs/vendor_api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,9 @@
+## v1.4 — 15th January 2024
+
+Minor Version Upgrade:
+
+Release API version `v1.4` to production. Please refer to the [API Reference](/api-docs/v1.4/reference) for details.
```

Comment:
What file is this referencing?

Reply:
It's just a reference to these endpoints: `https://www.apply-for-teacher-training.service.gov.uk/api-docs/v1.3/reference`